src/libs/actions/Session/index.ts

```diff
@@ -301,6 +304,7 @@ function signOutAndRedirectToSignIn(shouldResetToHome?: boolean, shouldStashSess
     // stashed session, then switch the account instead of completely logging out.
     if (shouldRestoreStashedSession && !shouldStashSession && hasStashedSession()) {
         if (CONFIG.IS_HYBRID_APP) {
+            isSwitchedAccount = true;
```

Comment:
NAB - is it better if we put this after `HybridAppModule.switchAccount` so that we know it succeeded?